hogben/simulate.py

```diff
@@ -8,9 +10,32 @@
 import numpy as np
 
 
-def simulate_magnetic(sample, angle_times, scale=1, bkg=5e-7, dq=2,
-                      mm=True, mp=True, pm=True, pp=True, angle_scale=0.7,
-                      directbeam_path=None):
+def DB_path(instrument: str, polarised: bool = False) -> str:
+    """Returns the filepath of the correct direct beam file for the instrument being used
+
+    Args:
+        instrument: The name of the instrument the experiement will be performed on
+        polarised: If the experiment will be polarised
+
+    Returns:
+        str: The filepath of the correct direct beam file
+    """
+
+    non_pol_instr = {'OFFSPEC': 'OFFSPEC_non_polarised_old.dat',
+                     'SURF': 'SURF_non_polarised.dat',
+                     'POLREF': 'POLREF_non_polarised.dat'}.get(instrument, 'OFFSPEC_non_polarised_old.dat')
+
+    pol_instr = {'OFFSPEC': 'OFFSPEC_polarised_old.dat'}.get(instrument, 'OFFSPEC_polarised_old.dat')
+    if not polarised:
+        path = importlib_resources.files('hogben.data.directbeams').joinpath(non_pol_instr)
+    else:
+        path = importlib_resources.files('hogben.data.directbeams').joinpath(pol_instr)
+    return path
+
+
+def simulate_magnetic(sample, angle_times: np.ndarray, scale: float = 1.0, bkg: float = 5e-7, dq: float = 2,
```

Comment:
The type of `angle_times` being `np.ndarray` doesn't tell you much, maybe some more description in the docstring about what 'points and times' are?